tests/io/v3/vm/credentials/test_credentials_schema.py

```diff
@@ -0,0 +1,84 @@
+'''
+Testing the Credentials schemas
+'''
+import pytest
+from marshmallow.exceptions import ValidationError
+
+from tenable.io.v3.vm.credentials.schema import (CredentialsCreateSchema,
+                                                 CredentialsEditSchema)
+
+PERMISSION_DATA = [(
+    'group', 64, '00000000-0000-0000-0000-000000000000'
+), (
+    'group', 'use', '00000000-0000-0000-0000-000000000000'
+), (
+    'group', 'edit', '00000000-0000-0000-0000-000000000000'
+), (
+    'user', 32, '00000000-0000-0000-0000-000000000000'
+)]
+
+PERMISSION_VALIDATED_DATA = [
```

Comment:
Can we make it more redable:

PERMISSION_VALIDATED_DATA = [
    {
        'grantee_uuid': '00000000-0000-0000-0000-000000000000',
        'type': 'group',
        'permissions': 64,
    },
    {
        'grantee_uuid': '00000000-0000-0000-0000-000000000000',
        'type': 'group',
        'permissions': 32,
    },
    {
        'grantee_uuid': '00000000-0000-0000-0000-000000000000',
        'type': 'group',
        'permissions': 64,
    },
    {
        'grantee_uuid': '00000000-0000-0000-0000-000000000000',
        'type': 'user',
        'permissions': 32,
    },
]